package.json

```diff
@@ -0,0 +1,81 @@
+{
+  "name": "smart-starter-app",
+  "version": "1.0.0",
+  "author": "Cerner Corporation, Cloud App Dev Platform",
+  "description": "== README",
+  "main": "./src/starter_app.js",
+  "style": "dist/css/starter_app.css",
+  "less": "src/less/manifest.less",
+  "license": "== LICENSE",
+  "homepage": "https://github.com/parthivbhagat/pb026393.github.io",
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:parthivbhagat/pb026393.github.io.git"
+  },
+  "dependencies":{
+    "fhirclient": "^0.1.9",
+    "jquery": "^2.1.4"
+  },
+  "scripts": {
+    "build": "npm run build-es5 && npm run build-webpack && npm run build-minify",
+    "build-doc": "jsdoc  --package package.json -d doc ./src/**/*.js",
+    "build-es5": "babel src --out-dir dist",
+    "build-webpack": "webpack --progress --colors --display-error-details",
+    "build-minify": "npm run build-minify-js && npm run build-minify-css",
+    "build-minify-js": "uglifyjs dist/js/starter_app.js > dist/js/starter_app.min.js",
+    "build-minify-css": "lessc --clean-css dist/css/starter_app.css dist/css/starter_app.min.css",
+    "lint": "npm run lint-js && npm run lint-css",
+    "lint-js": "jshint src/*.js --config .jshintrc; exit 0",
```

Comment:
let's remove this `exit 0` to cause the build to fail if there is any lint issue.